config/config.yaml

```diff
@@ -6,6 +6,7 @@ indexerConfig:
   luceneConfig:
     commitDurationSecs: ${INDEXER_COMMIT_DURATION_SECS:-10}
     refreshDurationSecs: ${INDEXER_REFRESH_DURATION_SECS:-11}
+    enableFullTextSearch: ${INDEXER_ENABLE_FULL_TEXT_SEARCH:false}
```

Comment:
This still isn't quite right - this needs to be `:-false`

Reply:
oops. Done.